lib/devicetrust/native/device_windows.go

```diff
@@ -16,55 +16,63 @@ package native
 
 import (
 	"bytes"
+	"context"
 	"crypto/rsa"
 	"crypto/sha256"
 	"crypto/x509"
 	"encoding/base64"
+	"fmt"
 	"math/big"
 	"os"
 	"os/exec"
 	"os/user"
 	"path/filepath"
+	"strings"
+	"syscall"
+	"time"
 
 	"github.com/google/go-attestation/attest"
 	"github.com/gravitational/trace"
 	log "github.com/sirupsen/logrus"
+	"golang.org/x/sys/windows"
 	"google.golang.org/protobuf/types/known/timestamppb"
 
 	devicepb "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1"
 	"github.com/gravitational/teleport/lib/devicetrust"
 )
 
 const (
-	deviceStateFolderName  = ".teleport-device"
-	attestationKeyFileName = "attestation.key"
+	deviceStateFolderName        = ".teleport-device"
+	attestationKeyFileName       = "attestation.key"
+	credentialActivationFileName = "credential-activation"
 )
 
 // setupDeviceStateDir ensures that device state directory exists.
 // It returns the absolute path to where the attestation key can be found:
 // $CONFIG_DIR/teleport-device/attestation.key
-func setupDeviceStateDir(getBaseDir func() (string, error)) (akPath string, err error) {
+func setupDeviceStateDir(getBaseDir func() (string, error)) (akPath string, credentialActivationPath string, err error) {
```

Comment:
Suggestion: we could return only the base path and let the caller concat file names, or we could return a struct with all the paths we care about inside it. The latter is a bit more readable and supports new paths more cleanly. (It can also be nil for `return nil, err` cases.)